src/cinema.ts

```diff
@@ -0,0 +1,172 @@
+// deno-lint-ignore-file no-explicit-any adjacent-overload-signatures
+import { Json } from './beatmap.ts';
+import { jsonGet } from './general.ts';
+
+export class CinemaScreen {
+    json: Record<string, any> = {
```

Comment:
It would be preferable if the json type was strongly typed instead of `Record<string, any>`

Reply:
👍